next.config.js

```diff
@@ -1,4 +1,19 @@
+const { hostname } = require("os");
```

Comment:
```suggestion
```

don't need this, I'm guessing it was an accidental auto-import

Reply:
Removed, not sure where this came from